drivers/ieee802154/ieee802154_rf2xx.c

```diff
@@ -363,15 +405,22 @@ static int rf2xx_set_channel(struct device *dev, uint16_t channel)
 
 static int rf2xx_set_txpower(struct device *dev, int16_t dbm)
 {
-	uint8_t reg;
-
-	ARG_UNUSED(dbm);
-
-	/* TODO: Add look-up table
-	 * Now will max power
-	 */
-	reg = rf2xx_iface_reg_read(dev, RF2XX_PHY_TX_PWR_REG) & ~0x0f;
-	rf2xx_iface_reg_write(dev, RF2XX_PHY_TX_PWR_REG, reg);
+	if (dbm < RF2XX_OUTPUT_POWER_MIN) {
+		LOG_INF("TX-power %d dBm below min of %d dBm, using %d dBm",
+			    dbm,
```

Comment:
you seem to have still an issue with indentation. Can your editor apply linux code style?
indented parameters should be below last'(' + 1 space.

Reply:
Should be fixed now. Installed the VScode editorconfig extension.